### DISCUSSION TITLE
最近是不是更新了代码但是没有发包？几个依赖包都找不到...

### DISCUSSION THREAD

**Discussion by wwyx778** [alibaba/designable#201] [Q&A]
![image](https://user-images.githubusercontent.com/28632663/149940231-e03ec9f9-42b6-4c56-9464-3768dfe438e1.png)

**Comment by janryWang** [comment]
没发布过

**Reply by wwyx778** [reply]
发布下NPM？现在拉取最新代码有些依赖找不到。
